infra/src/main/kotlin/fi/fta/geoviite/infra/configuration/CachePreloadScheduler.kt

```diff
@@ -32,14 +32,21 @@ constructor(
     private fun scheduleReload() {
         val startTime = System.currentTimeMillis()
         logger.info("Preloading caches: geocodingContexts=$preloadGeocodingContexts planHeaders=$preloadPlanHeaders")
+        // First stage caches that can be loaded in parallel
         listOf(
                 { cachePreloadService.loadAlignmentCache() },
                 { cachePreloadService.loadLayoutCache() },
                 { if (preloadPlanHeaders) cachePreloadService.loadPlanHeaderCache() },
             )
             .parallelStream()
             .forEach { preload -> preload() }
-        if (preloadGeocodingContexts) cachePreloadService.loadGeocodingContextCache()
+        // Second stage caches that use the first stage and need to be loaded after them
+        listOf(
+                { cachePreloadService.loadLocationTrackSpatialCache() },
```

Comment:
Tälle voisi laittaa oman propsun, samassa hengessä kuin geocodingContextit, koska tätähän ei normipuolella (vielä) käytetä.

Reply:
Onko syytä olettaa että perusbäkkäripuolella ei tätä haluttaisi käyttää? 🤔 Jos tässä lämmittelyssä ei mene ikuisuuksia tai tuo kakku syö kaikkea muistia koskaan ikinä, niin mielestäni se voisi olla ihan OK tehdä sielläkin, koska sitten olisi yksi asia vähemmän tämän käyttöönoton tiellä sillä puolella. Plus silloin on vaarana että tuo kakku olisi hidas (mahdollisesti jopa wanhaa hakutapaa hitaampi) ennen kuin preload otetaan käyttöön, joka saattaisi ohjata toteutuksia väärään suuntaan jos silloin ei vaan muisteta että preload pitäisi napata päälle.

Reply:
Se on alle sekunnin latausaika joten ei siinä mielessä ongelma ja kyllä tuota voisi haluta käyttää myös main-puolella mutta ei vielä käytetä. Voisin ehkä tiketöidä tuon propsuttamisen/käyttöönoton tutkiskelun ja pistää nyt vain eteenpäin tällaisenaan.